[tooltip] fixed popper content announcing (#2541)
semrush/intergalactic: semcore/tooltip/CHANGELOG.md

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

## [16.0.11] - 2025-10-30

### Fixed

> - Tooltip popper content are not announced by SR in chrome.

Reviewer: ```suggestion
- The tooltip popper content is not announced by screen readers in Chrome.
```
